mapper/mapper_test.go

```diff
@@ -0,0 +1,88 @@
+package mapper
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+)
+
+func TestIsClassifiedByMappedCorrectly(t *testing.T) {
+
+	originalBody, err := ioutil.ReadFile("test_resources/annotations_isClassifiedBy_v2.json")
+	if err != nil {
+		panic(err)
+	}
+	expectedBody, err := ioutil.ReadFile("test_resources/annotations_isClassifiedBy_PAC.json")
+
+	if err != nil {
+		panic(err)
+	}
+	actualBody, _ := ConvertPredicates(originalBody)
+
+	log.Info(string(actualBody))
+	log.Info(string(expectedBody))
+
```

Comment:
I'm wondering if we need to log this every time?

Reply:
I left that in by accident